yaksh/management/commands/add_moderator.py

```diff
@@ -0,0 +1,37 @@
+'''
+   This command adds users to the moderator group with permissions to add, change and delete
+   the objects in the exam app.
+'''
+
+# django imports
+from django.core.management.base import BaseCommand, CommandError
+from django.contrib.auth.models import User, Group, Permission
+from django.contrib.contenttypes.models import ContentType
+from django.db.utils import IntegrityError
+
+# Yaksh imports
+from yaksh.models import Profile
+
+class Command(BaseCommand):
+    help = 'Adds users to the moderator group'
+
+    def add_arguments(self, parser):
+        # Positional arguments
+        parser.add_argument('usernames', nargs='+', type=str)
+
+    def handle(self, *args, **options):
+        try:
+            group = Group.objects.get(name='moderator')
+        except Group.DoesNotExist:
+            raise CommandError('Moderator Group does not exist')
+
+        for uname in options['usernames']:
+            try:
+                user = User.objects.get(username=uname)
+            except User.DoesNotExist:
+                raise CommandError('User "{0}" does not exist'.format(uname))
+            if user in group.user_set.all():
+                self.stdout.write('User "{0}" is already a Moderator'.format(uname))
+                continue               
```

Comment:
Why continue? Why unnecessarily add user to the group yet again? Couldn't the adding part be in an `else` condition?

Reply:
the ```continue``` command breaks out of that iteration of the loop and moves to the next iteration so that user is not added to the group and hence ```else``` is not required.